### ISSUE TITLE
Track the status of the demo site automatically

### ISSUE THREAD

**Issue by tuliren** [airbytehq/airbyte#4141]
## Tell us about the problem you're trying to solve
Our demo site runs into issues relatively frequently. We should have an automatic tracker for it, so that we can know it is down as soon as possible.

## Describe the solution you’d like
Maybe track it using https://www.atlassian.com/software/statuspage or https://uptimerobot.com/.

## Describe the alternative you’ve considered or used
N/A

## Additional context
N/A

## Are you willing to submit a PR?
Yes

**Comment by cgardens** [comment]
Agreed. What are some of the things we should check for? The easy one is to just hit the `/health_check` endpoint. I kinda think that'll be insufficient coverage though.

**Comment by jrhizor** [comment]
I think if we check 1) the static site is accessible 2) `/health` is accessible and 3) `list_connections` is available we would catch almost all issues.
